cmd/nvidia-dra-controller/main.go

```diff
@@ -154,14 +156,26 @@ func newApp() *cli.App {
 				}
 			}
 
+			sigs := make(chan os.Signal, 1)
+			signal.Notify(sigs, syscall.SIGTERM, syscall.SIGINT)
+
+			var imexManager *ImexManager
+			ctx, cancel := context.WithCancel(c.Context)
+			defer func() {
+				cancel()
+				if err := imexManager.Stop(); err != nil {
+					klog.Errorf("error stopping IMEX manager: %v", err)
```

Comment:
```suggestion
					klog.Errorf("Error stopping IMEX manager: %v", err)
```

Reply:
done

Reply:
oh I forgot linters don't like capitalized error messages

Reply:
Reverted this change, to comply with the linter

Reply:
WONT-DO

Reply:
Really? Did the linter throw an erro? I have always operated with returned errors must be lower case but logs must be upper case.

Reply:
Yeah I tried and it complains about upper case

Reply:
Can you point me where you are seeing the error?

When I run the following I don't have any lint errors:
```
$ make golangci-lint
golangci-lint run ./...
```

We've always operated such that log messages are capitalized and returned error messages are not (until they are printed as a log at the top of the stack -- which is capitalized). I'd like to stay consistent with this.

Reply:
https://github.com/NVIDIA/k8s-dra-driver/actions/runs/11481541055/job/31952466758#step:5:27

Reply:
```bash
  Running [/home/runner/golangci-lint-1.61.0-linux-amd64/golangci-lint run  -v --timeout 5m] in [/home/runner/work/k8s-dra-driver/k8s-dra-driver] ...
  Error: cmd/nvidia-dra-controller/imex.go:309:11: ST1005: error strings should not be capitalized (stylecheck)
  			return fmt.Errorf("Error deleting resource slice %s: %w", rs.Name, err)
  ```

Reply:
Maybe your `golangci-lint` or `go` versions are different from the one we have on CI

Reply:
That is not the correct line you are changing. That one *should* be lower case because it is a return statement. The line I#m requesting to change is the `klog` line which *should* be uppercase.